retirement_api/utils/ss_update_stats.py

```diff
@@ -76,9 +80,9 @@ def output_json(filepath, headings, bs_rows):
 def make_soup(url):
     req = requests.get(url)
     if req.reason != 'OK':
-        print "request to %s failed: %s %s" % (url,
```

Comment:
One side effect of making these changes is that the behavior changes from always printing the warning (here and in other changes below) to only sometimes, depending on how logging is configured:

1. If running standalone using the test settings, will these ever be printed, if the log threshold is `CRITICAL`?
2. If running standalone using some other settings, without any logger configured, I believe the user would see a message about no handlers being configured the first time, but none of the warning messages.
3. If running standalone with logging handler of `INFO`/`WARNING` (for example as is configured in cfgov-refesh), the messages will be printed.

Due to the way cfgov-refresh is configured, we could leave it this way, but it feels like a degradation to potentially hide these warning messages. One possible option would be to add some additional local settings file; or maybe add a comment to the README about use of logging for warnings. I'm not really sure - do you think this is worth worrying about or have any other ideas about how to mitigate the change?

Reply:
1. I had assumed that the test settings were only used for tests, not for running the project. Under what circumstances would one want to run it with test settings?
2. Oh, I see. I tried looking in other satellite apps to see how they have logging configured, but NONE of them does. I'll look into how to set up logging while running standalone.

I think it's worth spending a bit of time ensuring that someone running the app locally sees the logged warnings.

Reply:
When running this app standalone, using e.g. `manage.py runserver` [defaults to using the test settings](https://github.com/cfpb/retirement/blob/master/manage.py#L6). So ideally someone might have their own test settings but I can imagine using those to get started and then not expecting the logging behavior.